.prettierignore

```diff
@@ -24,3 +24,5 @@ yarn-error.log*
 dist
 .solid
 pnpm-lock.yaml
+
+CHANGELOG.md
```

Comment:
Does it need to be globbed to nested folders?

Reply:
It passes just like this... 🤷